src/sound/openal.c

```diff
@@ -26,9 +26,9 @@
 # undef ALC_API
 # define AL_LIBTYPE_STATIC
 # define ALC_LIBTYPE_STATIC
-# include <AL/al.h>
-# include <AL/alc.h>
-# include <AL/alext.h>
+# include "al.h"
+# include "alc.h"
+# include "alext.h"
```

Comment:
This breaks on the old MinGW makefile.

Reply:
True, that was already risky on macOS and if it breaks MinGW, it has to be fixed.

Reply:
Perhaps it would be better to include the full path and differentiate between the different macOS includes by using an `#ifdef` rather than relying on the include path provided by FindOpenAL?

Reply:
And in src/codegen_new/CMakeFiles/dynarec.dir/codegen_backend_x86-64.c and also respectively src/codegen_new/CMakeFiles/dynarec.dir/codegen_backend_x86.c and src/codegen_new/CMakeFiles/dynarec.dir/codegen_backend_arm64.c the include for string.h is missing, which breaks compiling with the new dynarec on macOS.